urban-dashboard/src/components/mapboxViewer/index.jsx

```diff
@@ -35,11 +35,14 @@ export class MapBoxViewer extends Component {
 
         const map = new mapboxgl.Map({
             container: 'mapbox-container',
-            renderingMode: '2D',
+            projection: "mercator",
             style: mapboxStyleUrl,
             center: [-99.676392, 39.106667], // Center of the USA
-            zoom: 4.8, // Adjust zoom level to fit the USA
-            zoomControl: true
+            zoom: 4, // Adjust zoom level to fit the USA
```

Comment:
this works better in embedded view

Reply:
If possible could we add an ability to pass in zoom level as a query params? It will be very handy to fine-tune zoom levels according to the viewer's size. (ref. NIST interface). Not a necessary one but a nice to have one.